3-networks/README.md

```diff
@@ -128,3 +128,7 @@ and only the corresponding environment is applied.
 1. Run `./tf-wrapper.sh apply development`
 
 If you received any errors or made any changes to the Terraform config or any `.tfvars`you must re-run `./tf-wrapper.sh plan <env>` before run `./tf-wrapper.sh apply <env>`
+
+### Optional Variables
```

Comment:
If @atos-cit agrees with my feedback here https://github.com/terraform-google-modules/terraform-example-foundation/pull/239, I think we can remove this.

Reply:
Removed the section as agreed in #239 .